pulsar/consumer_partition.go

```diff
@@ -323,7 +323,7 @@ func (pc *partitionConsumer) internalSeekByTime(seek *seekByTimeRequest) {
 	cmdSeek := &pb.CommandSeek{
 		ConsumerId:         proto.Uint64(pc.consumerID),
 		RequestId:          proto.Uint64(requestID),
-		MessagePublishTime: proto.Uint64(uint64(seek.publishTime.Unix())),
+		MessagePublishTime: proto.Uint64(uint64(seek.publishTime.UnixNano() / 1e6)),
```

Comment:
Should we use a constant? `seek.publishTime.UnixNano() / int64(time.Millisecond)`

Reply:
Thanks @cckellogg will fix it